JigsawSwap/Network/Requests/AddJigsawRequest.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+class AddJigsawRequest: APIRequest {
+    var method: RequestType = .POST
+    var path: FirebasePath = .jigsaws
+    var parameters: [String : String] = [:]
+    var isAuthRequired: Bool = true
+    var body: Data? = nil
+    
+    init(_ jigsaw: Jigsaw) {
+        do {
+            body = try JSONEncoder().encode(jigsaw)
+        } catch {
+            print(error)
+        }
+    }
+    
+    init(_ jigsaws: [Jigsaw]) {
+        body = try? JSONEncoder().encode(jigsaws)
```

Comment:
`https://github.com/dominika-czarnecka/JigsawSwap/pull/7/files#diff-bcc03cf8f8da61e61532e1d2380b3af5R11` to masz do catch a tutaj try?, niekonsekwencja